js/form-validation.js

```diff
@@ -0,0 +1,55 @@
+/* const form = document.getElementById('contact-form');
+
+const validateEmail = (email) => {
+  var emailRegex = /^[a-z]+@[a-z0-9-]+\.[a-z0-9-.]+$/;
+  if (!emailRegex.test(email)) {
+    return false;
+  } else {
+    return true;
+  }
+}; */
+/* form.addEventListener('submit', (event) => {
+  // stop form submission
+  event.preventDefault();
+
+  const email = form.elements['email'];
+  if (validateEmail(email)) {
+    email.setCustomValidity(
+      'Please enter a valid email address and in lowercase letters.'
+    );
+
+    //display error message here
+  } else {
+    email.setCustomValidity('');
+    form.submit();
+  }
+}); */
+
+// const form = document.getElementById('contact-form');
+/* form.addEventListener('submit', (event) => {
+  // stop form submission
+  event.preventDefault();
+});
+
+const emailEl = document.querySelector('#email');
+form.addEventListener('submit', (event) => {
+  // prevent the form from submitting
+  event.preventDefault();
+});
+const isEmailValid = (email) => {
+  var emailRegex = /^[a-z]+@[a-z0-9-]+\.[a-z0-9-.]+$/;  return re.test(email);
+}; */
+
+/* const checkEmail = () => {
+  let valid = false;
+  const email = emailEl.value.trim();
+  if (!isRequired(email)) {
+    showError(emailEl, 'Email cannot be blank.');
+  } else if (!isEmailValid(email)) {
+    showError(emailEl, 'Email is not valid.');
+  } else {
+    showSuccess(emailEl);
+    valid = true;
+  }
+  return valid;
+}; */
```

Comment:
- [ ] Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. [You can read more about the Best Practices here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)